### DISCUSSION TITLE
How to run javajme on retroarch?

### DISCUSSION THREAD

**Discussion by Shimmer2** [SquirrelJME/SquirrelJME#121] [RetroArch (RatufaCoat)]
How to run javajme on retroarch?

**Comment by XerTheSquirrel** [comment]
Hi!

SquirrelJME on RetroArch is still a work in progress and should be available later this year.

**Reply by Shimmer2** [reply]
Thank you very much for your reply! Run java game of flip phone on Mac! It's exciting! expect!

**Comment by nikhilCad** [comment]
Is it available yet

**Reply by XerTheSquirrel** [reply]
Not currently yet, I am actively working on it and it is much to do.

**Comment by huaimogu** [comment]
Now?

**Reply by XerTheSquirrel** [reply]
Not yet.

**Comment by dixtdf** [comment]
RetroArch recently hit the AppStore and it would make a lot of sense to have RetroArch compatibility!

**Reply by XerTheSquirrel** [reply]
This is currently being worked on!

**Reply by dixtdf** [reply]
I'm looking forward to your bombshell.（￣︶￣）↗

**Reply by nvcuong1312** [reply]
I hope it will support internet m(_ _)m
